backend/Tasque.Core.Common/Security/SecurityHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+using System;
+using System.Security.Cryptography;
+
+namespace Tasque.Core.Common.Security
```

Comment:
A common project is a description of entities in our project. I don't know if you can write security logic here. Will be better to move this logic to DLL

Reply:
imo entities should be in dal and not common, but since they're in common I left validators with them

Reply:
These methods do not interact with the database or any critical data but provide utility to transform passwords. Maybe CryptographyHelper would be better name for this